AppBundles/DataCheckerPlugin/DataCheckerAutomation.cs

```diff
@@ -144,11 +178,17 @@ private void ExtractDrawingList()
                            !lowName.Contains(oldVersionMask);
                 })
                 .Select(path => path.Substring(index))
+                .OrderBy(path => System.IO.Path.GetFileName(path), new DefaultDocComparer(fileName))
                 .ToArray();
 
+            LogTrace("DEFAULT drawing is: " + (drawings.Count()>0 ? drawings[0] : null));
+
             SaveAsJson(drawings, "drawings-list.json"); // the file name must be in sync with activity definition
 
-            AddMessage($"Found {drawings.Length} drawings", Severity.Info);
+            AddMessage($"Found {drawings.Count()} drawings", Severity.Info);
+
+            foreach (var (d, i) in drawings.Select((v, i) => (v, i)))
```

Comment:
BTW - in latest C# no need to do the extra `Select`, just do the destruction:
https://github.com/Developer-Autodesk/forge-configurator-inventor/blob/4c36d4d963e35252fad76d7de60f8f9b6e00cf8a/WebApplication/Utilities/Collections.cs#L36

Reply:
but 'dict' is array not dictionary in this case

Reply:
I want to print index + value

Reply:
Argh! You are right, I misunderstood it completely 😁